examples/apps/collaborative-textarea/tests/collaborativetext.test.ts

```diff
@@ -64,6 +64,16 @@ describe("collaborativetext", () => {
 
         setText(0, "hello");
 
+        await page.waitForFunction( async () => {
+                const divs = document.getElementsByClassName("text-area");
+                const textAreaElements = divs[0].getElementsByTagName("textarea");
+                const textarea = textAreaElements[0] as HTMLTextAreaElement;
+                if (textarea) {
+                    return textarea.value.includes("hello");
+                }
+            }, { timeout: 1000 }
```

Comment:
It would be useful to leave some mark somewhere (console?) that timeout was hit - it would simplify debugging for next poor guy who wonders why his/her test failed

Reply:
Jest should show the error nicely when this times out